airflow/ui/src/components/DataTable/DataTable.tsx

```diff
@@ -106,87 +110,34 @@ export const DataTable = <TData,>({
     onStateChange: handleStateChange,
     rowCount: total,
     state: initialState,
+    ...extra,
```

Comment:
Doesn't this project typically use the name `rest` for spreads? Keep that consistency going?
```suggestion
    ...rest,
```